messages/rule-crashed.txt

```diff
@@ -0,0 +1,21 @@
+ESLint encountered an uncaught exception from rule "<%- ruleName %>".
+
+<% if (pluginName) { %>
+The rule in question comes from the plugin "eslint-plugin-<%- pluginName %>". You should open an issue on that plugin's issue tracker. Be sure to include your configuration, file source code, and the error message and stack trace.
+<% } else { %>
+This is a core rule, so this exception may indicate a bug in ESLint. Please open an issue in our issue tracker, using the following information in addition to your exact source code and configuration.
```

Comment:
@BYK Definitely, I just wanted to get something out there and I wasn't sure (at the time) how to detect a custom parser. @ilyavolodin was kind enough to dig up the PR showing where I can find it.